crates/devices/virtio-blk/src/defs.rs

```diff
@@ -1,47 +0,0 @@
-// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-//
-// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
-
```

Comment:
I don't think all the constants here are defined in virtio-bindings as well (for example SECTOR_SHIFT).